docker/owlbot/java/cloudbuild.yaml

```diff
@@ -17,7 +17,7 @@ steps:
     entrypoint: '/bin/sh'
     args:
       - '-c'
-      - 'git log -1 --format="%s%n%n%b%nSource-Link: https://github.com/googleapis/synthtool/commit/%H" > post-processor-changes.txt'
```

Comment:
I haven't heard the same request from Java or NodeJs people, so please make this change to python's cloudbuild.yaml only.

Reply:
Done.